docs/docs/Coding-Standards.md

```diff
@@ -0,0 +1,19 @@
+---
+id: Coding-Standards
+title: Coding Standards
+---
+
+These are some of the standards we choose to follow in this project. These might change if we agree on standards for javascript development within the community.
+
+### Code Comments
+
+We follow the Comments guidelines from [this style guide](https://prettier.io/playground/).
+
+### Code Formatter
+
+We use [Prettier](https://prettier.io/playground/) to help with code readability.
```

Comment:
Can we also add a description about using [Eslint](https://eslint.org/) as a code linter for the project, I saw usage of `Eslint` in #75 issue

Reply:
@Aaishpra As there is no PR for #75 , this can be added later through a first timer issue. What say ?

Reply:
I understand your point about first-timer issues but we don't want all the issues to be documentation related for the first-timers (considering the point we already have a lot of documentation related issues in this project when even the MVP is not complete) and since there is no PR for #75 how come we are adding usage of Prettier in the wiki (there is no PR for prettier too if not for Eslint)

Reply:
>but we don't want all the issues to be documentation related for the first-timers (considering the point we already have a lot of documentation related issues in this project when even the MVP is not complete)

@Aaishpra documentation issues are in no way blocking the completion of MVP. So I don't think we need to worry about the number of documentation issues rising. 

>how come we are adding usage of Prettier in the wiki (there is no PR for prettier too if not for Eslint)

Even I don't know why it is there in the wiki. @isabelcosta Can you please give your input here.

Reply:
Feel free to remove that if you think that Prettier is not being used. I would also suggest avoiding changing too much in this PR, and make this only about moving the wiki to the repository and leave the documentation fixes to another PR.

Reply:
Documentation might have been written initially with a plan for the project in mind, and then the plan did not get executed. So feel free to change this line to a more accurate version of the project.

Reply:
Thanks @isabelcosta 🙂